client/src/document/organisms/toc/index.tsx

```diff
@@ -5,8 +5,23 @@ import { Toc } from "../../../../../libs/types/document";
 import { useFirstVisibleElement } from "../../hooks";
 import { useGleanClick } from "../../../telemetry/glean-context";
 import { TOC_CLICK } from "../../../telemetry/constants";
+import { useLocale } from "../../../hooks";
+import { DEFAULT_LOCALE } from "../../../../../libs/constants";
+
+const DEFAULT_TITLE = {
+  "en-US": "In this article",
+  es: "En este artículo",
+  fr: "Dans cet article",
+  ja: "この記事では",
+  ko: "이 문서에서는",
+  "pt-BR": "Neste artigo",
+  ru: "В этой статье",
+  "zh-CN": "在本文中",
+  "zh-TW": "在本文中",
```

Comment:
@mdn/yari-content-zh Would you mind taking a look if these translations are correct?

PS: Please note that this is a one-off UI localization, and currently we do not accept any community PRs for UI localization.

Reply:
LGTM. Thank you @caugner!

Reply:
Thank you @yin1999.